src/client/components/AddMeal/AddMeal.css

```diff
@@ -0,0 +1,17 @@
+.flex-container {
+  display: flex;
```

Comment:
Same here. Name must descript what it styles. Function describe what they do. Components describe what they contain or what they do. 1 thing only